Dockerfile

```diff
@@ -36,6 +36,10 @@ ARG RUSTC_WRAPPER=cachepot
 ENV AWS_REGION=eu-central-1
 ENV CACHEPOT_S3_KEY_PREFIX=cachepot
 ARG CACHEPOT_BUCKET=neon-github-dev
+ENV PAGESERVER_CONFIG="broker_endpoint='http://storage_broker:50051'\n"\
+"pg_distrib_dir='/usr/local/'\n"\
+"listen_pg_addr='0.0.0.0:6400'\n"\
+"listen_http_addr='0.0.0.0:9898'\n"
```

Comment:
Since we don't really set `PAGESERVER_CONFIG` from the outside (and we mount a volume with config file), I think we can move these part of code below (to `RUN mkdir -p /data/.neon/ && \` ...), and just dump default config to the config file right away. 

```suggestion
```

Reply:
[b786f32](https://github.com/neondatabase/neon/pull/7766/commits/b786f32eeb62455b1e463f358c5759167475c2e0)